tests/plotting/test_plotter.py

```diff
@@ -394,6 +394,35 @@ def test_plotter_meshes(sphere, cube):
     assert len(pl.meshes) == 2
 
 
+def test_multi_block_color_cycler(verify_image_cache):
+    """Test passing a custom color cycler"""
+    verify_image_cache.windows_skip_image_cache = True
```

Comment:
We can remove `verify_image_cache` :)
```suggestion
def test_multi_block_color_cycler():
    """Test passing a custom color cycler"""
```